src/web/browsers/browsers-config.helper.ts

```diff
@@ -80,6 +80,10 @@ const getExtendedBrowsersConfigs = (config, commandArgs): any => {
     ];
   }
 
+  if (typeof config.chromeOptions !== 'undefined' && config.chromeOptions.length > 0) {
```

Comment:
In this case, it will return true if `chromeOptions` is a string.

```
const config = { chromeOptions: 'janek' }
typeof config.chromeOptions !== 'undefined' && config.chromeOptions.length > 0 // true
```